Engine/ParametersMenuModel.cs

```diff
@@ -83,21 +83,22 @@ public void ParameterizeOnParent()
         public bool IsParameter => member is ParameterMemberData;
         public bool MultiSelect => source?.Length != 1;
 
-        public bool TestPlanLocked
+        public bool TestPlanAllowsEdit
         {
             get
             {
-                var plan2 = source
-                    .Select(step => step is TestPlan plan ? plan : step.GetParent<TestPlan>()).FirstOrDefault();
-                return (plan2?.IsRunning ?? false) || (plan2?.Locked ?? false);
+                return source
+                    .Select(static step => step as TestPlan ?? step.GetParent<TestPlan>())
+                    .FirstOrDefault(static x => x != null)
+                    ?.AllowEdit ?? false;
             }
         }
```

Comment:
Maybe:
1. Undo the rename
2. Create a new property, `TestPlanAllowsEdit` with the current code
3. Update TestPlanLocked implementation:
```csharp
public bool TestPlanLocked => !TestPlanAllowsEdit;
```